services/node_manager/src/main/java/com/futurewei/alcor/nodemanager/service/implement/NodeFileLoader.java

```diff
@@ -96,11 +96,8 @@ private NodeInfo parseNodeObject(JSONObject nodeJson) throws InvalidDataExceptio
         String mac = (String) nodeJson.get(NodeManagerConstant.JSON_MAC1);
         String veth = (String) nodeJson.get(NodeManagerConstant.JSON_VETH1);
         int gRPCServerPort = NodeManagerConstant.GRPC_SERVER_PORT;
-        String unicastTopic = (String) nodeJson.get(NodeManagerConstant.UNICAST_TOPIC);
-        String multicastTopic = (String) nodeJson.get(NodeManagerConstant.MULTICAST_TOPIC);
-        String groupTopic = (String) nodeJson.get(NodeManagerConstant.GROUP_TOPIC);
         try {
-            node = new NodeInfo(id, name, ip, mac, veth, gRPCServerPort, unicastTopic, multicastTopic, groupTopic);
+            node = new NodeInfo(id, id, ip, mac, veth, gRPCServerPort);
```

Comment:
@pkommoju you may need to verify this change in the node manager.

Reply:
If you are talking about id to name change, this is bug here. NMM was unable to query/delete/update any node whose name and id were not the same of the nodes created by file upload, I fixed it a few commits ago.

Reply:
Sorry for making this bug. I have fixed it.

Reply:
+1